packages/newspringchurchapp/src/prayer/AddPrayer/AddPrayerCard/AddPrayerCard.js

```diff
@@ -0,0 +1,59 @@
+import React, { memo } from 'react';
+import { View } from 'react-native';
+import PropTypes from 'prop-types';
+import {
+  BodyText,
+  styled,
+  Button,
+  Card,
+  CardContent,
+  PaddedView,
+  Avatar,
+  H3,
+} from '@apollosproject/ui-kit';
+
+const StyledCardContent = styled({
+  alignItems: 'center',
+})(CardContent);
+
+const TitleView = styled(({ theme }) => ({
+  marginTop: theme.sizing.baseUnit * 0.5,
+  marginBottom: theme.sizing.baseUnit * 2,
+}))(View);
+
+const AddPrayerCard = memo(({ imgSrc, title, description, ...props }) => (
+  <Card>
+    <StyledCardContent>
+      {/* TODO: margin will be fixed pending core #667 */}
```

Comment:
Stale comment?

Reply:
no that needs to stay until we update to 0.8.4 or whenever it gets merged into core.

PS hi @conrad-vanl!